packages/popup/src/pages/Home/index.js

```diff
@@ -1,14 +1,46 @@
-const HomePage = () => {
+import Button from '@fluent-wallet/component-button'
+import {useTranslation} from 'react-i18next'
+import {HomeNav} from '../../components'
+import {
+  CurrentAccount,
+  CurrentNetwork,
+  CurrentDapp,
+  TokenList,
+} from './components'
+
+function Home() {
+  const {t} = useTranslation()
+  const onLock = () => {}
   return (
-    <div className="App">
-      <header className="App-header">
-        <button onClick={() => open(location.href)}>open</button>
-        <p className="p-9">
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-      </header>
+    <div className="flex flex-col bg-bg h-full">
+      <button onClick={() => open(location.href)}>open</button>
+      <HomeNav onLock={onLock} />
```

Comment:
What is the purpose of onLock  props here? Better remove it ?

Reply:
removed